drift/example/main.dart

```diff
@@ -5,18 +5,67 @@ import 'package:drift/native.dart';
 
 part 'main.g.dart';
 
+class TodoCategories extends Table {
+  IntColumn get id => integer().autoIncrement()();
+  TextColumn get name => text()();
+}
+
 class TodoItems extends Table {
   IntColumn get id => integer().autoIncrement()();
   TextColumn get title => text()();
   TextColumn get content => text().nullable()();
+  IntColumn get categoryId => integer().references(TodoCategories, #id)();
+
+  TextColumn get generatedText => text().nullable().generatedAs(
+      title + const Constant(' (') + content + const Constant(')'))();
+}
+
+abstract class TodoCategoryItemCount extends View {
```

Comment:
Could you add views to `drift/test/data/tables/todos.dart` as well? I would like to see a test ensuring that the `CREATE VIEW` statement matches what we expect (probably in `schema_test.dart`).